internal/overlord/servstate/handlers.go

```diff
@@ -606,7 +606,12 @@ func (s *serviceData) stop() error {
 			logger.Noticef("Cannot send SIGTERM to process: %v", err)
 		}
 		s.transition(stateTerminating)
-		time.AfterFunc(killWait, func() { logError(s.terminateTimeElapsed()) })
+
+		gracePeriod := killWait
```

Comment:
Assuming we're calling the plan field "grace-period", why don't we rename this `defaultGracePeriod` to match?